vllm/model_executor/models/mamba.py

```diff
@@ -38,10 +38,12 @@ class MambaDecoderLayer(nn.Module):
     def __init__(self,
                  config: MambaConfig,
                  cache_config: Optional[CacheConfig] = None,
-                 quant_config: Optional[QuantizationConfig] = None) -> None:
+                 quant_config: Optional[QuantizationConfig] = None,
+                 is_lora_enabled: Optional[bool] = False) -> None:
         super().__init__()
         self.config = config
         self.is_falcon_mamba = config.model_type == "falcon_mamba"
+        self.is_lora_enabled = is_lora_enabled
```

Comment:
Will you implement Lora support for `MambaForCausalLM` in this PR?

Reply:
Since I had to do changes in MambaMixer that is being used by both Jamba and Mamba, I made the required changes for Mamba as well

Reply:
@DarkLight1337  I think this PR just wan to  update Jamba lora module